docs/data-sources/node.md

```diff
@@ -50,32 +40,31 @@ data "wallarm_node" "example" {
 
 ## Argument Reference
 
-`filter` - (Required) Filters set in the `key=value` format used to look up for WAF node details. Possible keys:
+`filter` - (**required**) filters set in the `key=value` format used to look up for Wallarm node details. Possible keys:
 
-- `uuid` - (Optional) WAF node UUID.
-- `hostname` - (Optional) WAF node name.
-- `type` - (Optional) WAF node type. Can be: `cloud_node` for cloud WAF nodes, `node` for regular WAF nodes.
-- `enabled` - (Optional) Indicator of the WAF node status. Can be: `true` for enabled WAF nodes and `false` for disabled WAF nodes.
+- `uuid` - (optional) Wallarm node UUID.
+- `hostname` - (optional) Wallarm node name.
+- `enabled` - (optional) indicator of the Wallarm node status. Can be: `true` for enabled Wallarm nodes and `false` for disabled Wallarm nodes.
 
-To get details on all created WAF nodes, specify an empty set of the filters (`filter {}`).
+To get details on all created Wallarm nodes, specify an empty set of the filters (`filter {}`).
 
 ## Attributes Reference
 
-`nodes` - WAF node attributes in the `key=value` format. Possible keys:
-
-- `id` - Internal WAF node ID.
-- `hostname` - WAF node name.
-- `type` - WAF node type. Can be: `cloud_node` for cloud WAF nodes, `node` for regular WAF nodes.
-- `uuid` - WAF node UUID.
-- `enabled` - Indicator of the WAF node status. Can be: `true` for enabled WAF nodes and `false` for disabled WAF nodes.
-- `client_id` - ID of the client installed the WAF node.
-- `active` - Node syncronisation status
-- `instance_count` - Number of instances with the installed WAF node. Only for the `cloud_node` type.
-- `active_instance_count` - Number of active instances with the installed WAF node. Only for the `cloud_node` type.
-- `token` - WAF node token. Only for the `cloud_node` type.
-- `requests_amount` - Number of requests processed by the WAF node.
-- `ip` - IP address of the WAF node at the time of the last synchronization.
-- `proton` - Installed `proton.db` version. Can be an integer value or `null`.
-- `lom` - Installed `lom` version. Can be an integer value or `null`.
+`nodes` - Wallarm node attributes in the `key=value` format. Possible keys:
+
+- `id` - internal Wallarm node ID.
+- `hostname` - Wallarm node name.
+- `type` - Wallarm node type. Can be: `cloud_node` for cloud Wallarm nodes, `node` for regular Wallarm nodes. See [node types description](https://docs.wallarm.com/3.6/user-guides/nodes/nodes/#filtering-the-nodes). Note that `CDN` node is not supported.
```

Comment:
1. you are referring the user to the old documentation (the link will be deprecated very soon since we stop 3.6 support in November). the user will be confused. describe what is wrong with the "node" type right here
2. "cloud Wallarm nodes" - we do not use this term now, Wallarm nodes is the correct one